CONTRIBUTING.rst

```diff
@@ -1,47 +1,179 @@
 .. _contributing:
 
+How to help?
+=============
+
+* If you have a usage question : create a question on `neurostars <https://neurostars.org/>`_ with tag nilearn
+
+* You think you discovered a bug ? Create `an issue <https://github.com/nilearn/nilearn/issues>`_
+  including a minimal runnable example to showcase it (preferably using Nilearn data).
+
+* If you have ideas for new features, check it's in the scope of the project
+  and feel free to open an issue to discuss it.
+
+* You think documentation can be improved, you can directly send proposed
+  improvements in `a pull request <https://github.com/nilearn/nilearn/pulls>`_.
+
+* Want to contribute code ? Thank you very much! For new features, please be
+  sure to create an issue first, to discuss whether it can be included and its
+  specifications. If you have time to help dealing with known issues check
+  `Good first issues <https://github.com/nilearn/nilearn/labels/Good%20first%20issue>`_
+  to get started, `known bugs <https://github.com/nilearn/nilearn/labels/Bug>`_
+  or `proposed enhancements <https://github.com/nilearn/nilearn/labels/Enhancement>`_.
+  In any case, before sending code, be sure to ** read and follow the
+  contribution guide below **. Otherwise we might ask quite a lot of revisions
```

Comment:
Wrong rendering

Reply:
try without spaces inside the `**`: `**abc**` instead of `** abc **`